### PR TITLE
chore(cli): use upload api and update documentation

### PR DESCRIPTION
**Pull request by etnoy**
* Don't manually call fetch for the upload file
* Streamline the upload code
* Fix the include hidden option
* Improve documentation
* Bump to version 2.0.7

### CODE REVIEW

File: cli/src/commands/upload.command.ts [hunk 1]
```diff
@@ -48,28 +49,18 @@ class Asset {
     let sidecarData: Blob | undefined = undefined;
     try {
       await access(sideCarPath, constants.R_OK);
-      sidecarData = new File([await fs.openAsBlob(sideCarPath)], basename(sideCarPath));
+      sidecarData = await fs.openAsBlob(sideCarPath);
```
Comment (jrasm91) [inline cli/src/commands/upload.command.ts:52]: Why doesn't this need to be a new file, but the asset data does?

**Review by jrasm91 — APPROVED** [review]
LGTM. Did you test uploading a sidecar file?

File: cli/src/commands/upload.command.ts (hunk 1 above)
Comment (etnoy) [reply cli/src/commands/upload.command.ts:52]: After testing I reverted this.

**Comment by benmccann** [comment]
Cool. Thanks! This was something I noticed and was on my list too 😆  Something that would be good to confirm if you didn't already is that the axios and fetch clients use the same date serialization

**Comment by etnoy** [comment]
> Cool. Thanks! This was something I noticed and was on my list too 😆 Something that would be good to confirm if you didn't already is that the axios and fetch clients use the same date serialization

You don't want to know how long I fought with the axios sdk to get uploads working. In the end I had to concede to do manual http calls instead. 

With fetch, it just worked. Thanks for helping me with it

**Pull request merged**